src/PCLStorage.FileSystem.Desktop/DesktopFileSystem.cs

```diff
@@ -25,8 +25,11 @@ public IFolder LocalStorage
 #if ANDROID
                 var localAppData = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
 #elif IOS
-                var documents = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
-                var localAppData = Path.Combine(documents, "..", "Library");
+				var documents = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
It looks like you went from spaces to tabs here.  Can you change back to using spaces for indentation?